package.json

```
@@ -7,7 +7,7 @@
     "dev": "next dev -p 8080 ",
     "preprod": "NEXT_PUBLIC_API_URL=http://graphql-api.pre-prod.replay.prod/v1/graphql NEXT_PUBLIC_DISPATCH_URL=ws://agent.pre-prod.replay.prod next dev -p 8080",
     "export": "next build && next export -o dist",
-    "test": "jest",
+    "test": "jest -t syntax-highlighting",
```

Please revert this change.
```suggestion
    "test": "jest",
```

---

Yeah, forgot to revert back before pushing the changes, extremely sorry for that, will make the change.